e2e-pw/src/oss/poms/modal/looker-3d-controls.ts

```diff
@@ -21,4 +23,33 @@ export class Looker3DControlsPom {
       .getByTestId("toggle-looker-3d-render-preferences")
       .click();
   }
+
+  async waitForAllAssetsLoaded() {
+    await this.page.waitForFunction(
+      (SUCCESS_MSG_INJECTED) => {
+        const logs = document.querySelector(
+          "[data-cy=looker3d-logs-action-bar]"
+        );
+        return logs?.textContent === SUCCESS_MSG_INJECTED;
+      },
+      SUCCESS_MSG,
+      { timeout: 10000 }
+    );
+    // takes a bit of time for 3d assets to mount after load
+    // todo: figure out if we can emit event on canvas paint
+    // eslint-disable-next-line playwright/no-wait-for-timeout
+    await this.page.waitForTimeout(150);
```

Comment:
How confident are you that this won't result in a flaky test?

Reply:
I wrote that line with an aching heart :( 
Our loading manager emits "load" events but not "paint" events... The GPU is a black box and AFAIK can't set up event listeners for its operations like draw calls.

----
Only way to know is see if it causes any flakiness. Seems good so far.